web-components/src/components/cards/OrderSummaryCard/OrderSummaryCard.Content.tsx

```diff
@@ -0,0 +1,112 @@
+import { useState } from 'react';
+import { EditButtonIcon } from 'web-components/src/components/buttons/EditButtonIcon';
+import { DenomIconWithCurrency } from 'web-components/src/components/DenomIconWithCurrency/DenomIconWithCurrency';
+import { CURRENCIES } from 'web-components/src/components/DenomIconWithCurrency/DenomIconWithCurrency.constants';
+import { EditableInput } from 'web-components/src/components/inputs/new/EditableInput/EditableInput';
+import { SupCurrencyAndAmount } from 'web-components/src/components/SupCurrencyAndAmount/SupCurrencyAndAmount';
+import { Title } from 'web-components/src/components/typography';
+
+import { CRYPTO_TOOLTIP_TEXT } from './OrderSummaryCard.constants';
+import { OrderProps, PaymentMethod } from './OrderSummaryCard.types';
+import { OrderSummmaryRowHeader } from './OrderSummmaryCard.RowHeader';
+
+export function OrderSummaryContent({
+  order,
+  currentBuyingStep,
+  paymentMethod,
+  onClickEditCard = () => {},
+}: {
+  order: OrderProps;
+  currentBuyingStep: number;
+  paymentMethod: PaymentMethod;
+  onClickEditCard?: () => void;
+}) {
+  const { projectName, currency, pricePerCredit, credits } = order;
+  const [creditsAmount, setCreditsAmount] = useState(credits);
+  return (
+    <div className="grid grid-cols-[75px_1fr] sm:grid-cols-[90px_1fr] max-w-full w-full pr-15 sm:px-[20px] pb-[30px] items-center sm:max-w-[330px]">
+      <Title
+        variant="h5"
+        className="col-span-2 text-base font-black font-['Muli'] mt-0 sm:mt-[30px] mb-5 sm:mb-[15px]"
```

Comment:
those styles are already handled by `Title` component
```suggestion
        className="col-span-2 text-base mt-0 sm:mt-[30px] mb-5 sm:mb-[15px]"
```